org.mwc.debrief.core/src/org/mwc/debrief/core/actions/DragFeature.java

```diff
@@ -263,21 +299,13 @@ public void doMouseDrag(final org.eclipse.swt.graphics.Point pt,
 
         // remember where we are
         _lastPoint = new java.awt.Point(pt.x, pt.y);
-        final WorldLocation newLocation = new WorldLocation(_myCanvas
-            .getProjection().toWorld(_lastPoint));
-
-        // now work out the vector from the last place plotted to the current
-        // place
-        final WorldVector offset = newLocation.subtract(_lastLocation);
+        
+         _myCanvas.getCanvas().redraw();
+         _myCanvas.getCanvas().update();
+       
 
-        // draw new track
-        drawHere(gc, offset);
-
-        // remember the last location
-        _lastLocation = newLocation;
-        // ok, let's ditch the GC

```

Comment:
Hey, @G - I know we no longer call this code, but `fc.dispose()` looks like a mistake.  We only create it once, so we _should_ get `is disposed()` calls if it's called in the future.

Never mind, we don't dispose it at all now.